app-modules/integration-virtu/src/VirtuAdapter.php

```
@@ -0,0 +1,217 @@
+            subscriptionExternalId: $this->correlationKey($link),
+            status: 'pending',
+            planExternalId: $price->provider_price_id,
+            planSlug: $price->plan->slug,
+            quantity: $data->quantity,
+            endsAt: null,
+        )));
+
+        return $this->withBuyerIdentity($link->url, $billable);
+    }
+
+    public function purchaseCredits(Company|User $billable, Company $company, int $quantity, string $successUrl, string $cancelUrl): string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$successUrl'. (undefined)

(UnusedFormalParameter)